core/ustring.cpp

```diff
@@ -120,9 +120,13 @@ void String::copy_from(const char *p_cstr) {
 void String::copy_from(const CharType *p_cstr, int p_clip_to) {
 
 	int len = 0;
-	const CharType *ptr = p_cstr;
-	while (*(ptr++) != 0)
-		len++;
+
+	if (p_cstr) {
```

Comment:
We don't *need*, but I said it would be better. What if we added 100 LOCs before that `if (len == 0)` test? :) Functionally, if `p_cstr` is NULL, the method should abort right away IMO, that's cleaner.